drivers/at86rf215/at86rf215_o-qpsk.c

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (C) 2019 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_at86rf215
+ * @{
+ *
+ * @file
+ * @brief       Configuration of the MR-O-QPSK PHY on the AT86RF215 chip
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ * @}
+ */
+
+#include "at86rf215.h"
+#include "at86rf215_internal.h"
+
+#include "debug.h"
+
+/* IEEE Std 802.15.4g™-2012 Amendment 3
+ * Table 68d—Total number of channels and first channel center frequencies for SUN PHYs
+ * Table 68e—Center frequencies for the MR-O-QPSK PHY operating in the 868–870 MHz band
+ */
+
+// FIXME: center frequency / spacing is not uniform in sub-GHz band
```

Comment:
Left for another PR?

Reply:
> BTW looking trough the datasheet, any plans on supporting `basic mode`?

Wouldn't that already be achieved by disabling Auto-ACK and CSMA?

Reply:
Well IEEE 802.15.4 only supports one channel in the 868 MHz band.
Oddly enough RIOT defines `IEEE802154_CHANNEL_MAX_SUBGHZ` to 10, assuming the North American 900 MHz band

Reply:
Hmm can we but somewhere that only EU868 band is supported?

Reply:
I added a comment and reduced the number of channels in the sub-GHz band to 1